admin-dev/themes/new-theme/js/pages/translation-settings/ExportFormFieldToggle.js

```diff
@@ -81,6 +93,21 @@ export default class ExportFormFieldToggle {
     this.uncheck($themesType, $coreType);
     this.show($modulesValues);
     this.hide($themesValues, $coreValues);
+    this.subChoicesChanged();
+  }
+
+  subChoicesChanged() {
+    if (
+      ($coreType.is(':checked') && $coreCheckboxes.find(':checked').size() > 0)
```

Comment:
```suggestion
      ($coreType.prop('checked') && $coreCheckboxes.find(':checked').size() > 0)
```
Use property instead of jquery selector please :)

Reply:
I aproved the PR (lgtm) but I'll wait for this feedback to be treated before sending to QA 👍